frontend/src/app/workflows/mockup-viewer/mockup-viewer.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="h-100 d-flex flex-column">
+  <div class="btn-toolbar align-items-center p-2" role="toolbar" aria-label="Toolbar">
+    <div class="btn-group btn-group-sm me-auto" role="group" aria-label="Page controls">
+      <button type="button" class="btn btn-secondary btn-sm"
+              (click)="setFirst()"
+              [disabled]="currentDisplay === 'class'">
+        First
+      </button>
+      <button type="button" class="btn btn-secondary btn-sm"
+              (click)="previous()"
+              [disabled]="currentDisplay === 'class'">
+        Previous
+      </button>
+      <button type="button" class="btn btn-secondary btn-sm"
+              (click)="next()"
+              [disabled]="currentDisplay === 'class'">
+        Next
+      </button>
+      <button type="button" class="btn btn-secondary btn-sm"
+              (click)="setLast()"
+              [disabled]="currentDisplay === 'class'">
+        Last
+      </button>
```

Comment:
I believe this should be removed in favor of tabs that cleary show the diagram/page names. That way, one can select the one they want to see immediately instead of having to click through all the pages.

Reply:
Then why not offer both? I agree the buttons are useful for a "slideshow" kind of thing, but it would also be nice to see at a glance all the different pages/diagrams/files that exist. It also feels more in line with the scenario editor which also uses tabs (but does not need slideshow buttons because there is no defined order).

Reply:
I will try it out, my concern right now is, that the view becomes overloaded. But that can never be checked if not tried out.